ydb/core/protos/flat_scheme_op.proto

```diff
@@ -989,6 +993,13 @@ message TIndexDescription {
     repeated string DataColumnNames = 8;
     // DataSize + IndexSize of indexImplTable
     optional uint64 DataSize = 9;
+
+    // indexImplTable settings
+    oneof Partitions {
+        uint32 UniformPartitions = 10;
+        TExplicitPartitions PartitionAtKeys = 11;
```

Comment:
`Keys` is misleading here. Split points/boundaries are merely prefixes (dividing planes in the key space) and most often are not precise enough to be a full sized keys.
And we are not bound to any compatibility issues here, this being a private proto spec.

Reply:
Changed:

```diff
     oneof Partitions {
         uint32 UniformPartitions = 10;
-        TExplicitPartitions PartitionAtKeys = 11;
+        TExplicitPartitions ExplicitPartitions = 11;
     }
```

```diff
 message TExplicitPartitions {
-    repeated TSplitBoundary SplitBoundary = 1;
+    repeated TSplitBoundary SplitBoundaries = 1;
 }
```
 (it seems to me that repeatable field names should be plural to better express the semantics of a collection)